File: C and CPP Programs/solution2.c
#include<stdio.h>
int main()
{

    int wage_rate, days, salary;

    // taking input, wage rate per day
    printf("Enter the wage rate per day(in $): ");
    scanf("%d",&wage_rate);

    // taking input, number of days
    printf("Number of days worked: ");
    scanf("%d", &days);

    // calculating salary
    salary = wage_rate*days;

    // output the salary
    printf("Total salary: %d$\n",salary);

    return 0;
}

File: WindowUpdate/Solution2.py
while True:
    count = 0
    while True:
        num = int(input("Enter a number: "))
        if num == 0:
            break
       
        # count above 50
        if num>50:
            count = count + 1
    # print the Count of above 50
    print(f'Count of above 50 is: {count}')
    # taking input to continue or stop
    des = int(input("Enter '0' for exit, '1' for continue: "))
    if des == 0:
        break

    